quartz/plugins/transformers/ofm.ts

```diff
@@ -492,6 +492,22 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
                     "data-callout-metadata": calloutMetaData,
                   },
                 }
+
+                // Add callout-content class to callout body if it has one.
+                if (node.children.length > 1) {
+                  const contentData = node.children[1]
+                  node.children[1] = {
+                    data: {
+                      hProperties: {
+                        ...(contentData.data?.hProperties ?? {}),
+                        className: "callout-content",
+                      },
+                      hName: "div",
+                    },
+                    type: "blockquote",
```

Comment:
why is type blockquote here? seems like it should be a div and not use hName

Reply:
![image](https://github.com/jackyzha0/quartz/assets/8161064/ab1c4bcf-6bc6-42ab-8f2c-519989515adf)

Reply:
Changing it to `type: "html"` results in the following error:

![image](https://github.com/jackyzha0/quartz/assets/8161064/d1e1e2fc-23d2-4f17-9b8a-38461a9d4ed7)

Keep in mind that a callout is technically a `blockquote`.

Reply:
this is the inner child which isnt a blockquote

Reply:
we should do this above on L475 `node.children.splice(0, 1, ...blockquoteContent)` and wrap `blockquoteContent` with an element there